src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -913,4 +913,10 @@ This version used in: {2}</value>
   <data name="Install_version_0" xml:space="preserve">
     <value>Install version '{0}'</value>
   </data>
-</root>
+  <data name="AddMissingStatements" xml:space="preserve">
```

Comment:
Why do we need both of these resources?  Also, didn't we see a resources already added for this above?